src/private/Write-AzOpsLog.ps1

```diff
@@ -62,7 +62,4 @@ function Write-AzOpsLog {
             }
         }
     }
-
-    end { }
```

Comment:
Could we this block in, we'd like to have a consistent code structure across all functions?